packages/gatsby/src/utils/handle-flags.ts

```diff
@@ -0,0 +1,89 @@
+import _ from "lodash"
+import { isCI } from "gatsby-core-utils"
+import realTerminalLink from "terminal-link"
+import { IFlag } from "./flags"
+
+const terminalLink = (text, url): string => {
+  if (process.env.NODE_ENV === `test`) {
+    return `${text} (${url})`
+  } else {
+    return realTerminalLink(text, url)
+  }
+}
+
+const handleFlags = (
+  flags: Array<IFlag>,
+  configFlags: Array<Array<string>>,
```

Comment:
is configFlags an array, isn't it an object? We could convert string to all available keys if we want to
```suggestion
  configFlags: Record<string, boolean>
```